mlflow/utils/uri.py

```diff
@@ -27,29 +27,22 @@ def is_databricks_uri(uri):
     return scheme == 'databricks' or uri == 'databricks'
 
 
-def get_db_profile_from_uri(uri):
-    """
-    Get the Databricks profile specified by the tracking URI (if any), otherwise
-    returns None.
-    """
-    parsed_uri = urllib.parse.urlparse(uri)
-    if parsed_uri.scheme == "databricks":
-        return parsed_uri.netloc
-    return None
+def construct_db_uri_from_profile(profile):
+    if profile:
+        return 'databricks://' + profile
 
 
-def get_db_path_info_from_uri(uri):
+def get_db_info_from_uri(uri):
     """
-    Get additional prefix information specified by the tracking URI (if any), otherwise
+    Get the Databricks profile specified by the tracking URI (if any), otherwise
     returns None.
     """
     parsed_uri = urllib.parse.urlparse(uri)
     if parsed_uri.scheme == "databricks":
-        if parsed_uri.path:
-            if parsed_uri.path[0] == '/':
-                return parsed_uri.path[1:]
-            return parsed_uri.path
-    return None
+        parsed_path = parsed_uri.path.lstrip('/') or None
```

Comment:
what does `or None` do here?

Reply:
Previously, if the parsed path was '' it would return None, but if it was '/', it would return ''. To make it consistent, `or None` basically ensures that if the returned value is falsey (''), that it is converted into None.